looker_blueprints/run_query_and_download.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import sys
+import os
+import shipyard_utils as shipyard
+try:
+    import helpers
+except BaseException:
+    from . import helpers
+
+
+EXIT_CODE_QUERY_ERROR = 203
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--base-url', dest='base_url', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--client-secret', dest='client_secret', required=True)
+    parser.add_argument('--query-id', dest='query_id', required=False)
```

Comment:
query-id should be required. Also file_type, dest_file_name, dest_file_folder should be arguments here.